package.json

```diff
@@ -136,18 +136,33 @@
           "when": "resourceFilename =~ /[dD]ocker-[cC]ompose/",
           "command": "vscode-docker.compose.restart",
           "group": "docker"
+        },
+        {
+          "when": "explorerResourceIsFolder && explorerResourceIsRoot",
+          "command": "vscode-docker.configure",
+          "group": "docker"
         }
       ],
       "view/title": [
         {
           "command": "vscode-docker.explorer.refresh",
           "when": "view == dockerExplorer",
-          "group": "navigation"
+          "group": "navigation@1"
```

Comment:
Every other extension has refresh on the farthest right. The file explorer also has it on the right of the 'create' actions